common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2387,6 +2387,12 @@
     ],
     "sqlState" : "42604"
   },
+  "INVALID_EXECUTOR_HEARTBEAT_INTERVAL" : {
```

Comment:
Discussed offline with Bo and Xingbo. By adding a new error class for heartbeat config might be too much, as we could have a lot of config validations, this way may require us to add many different error classes.

A better way for doing this is by adding a more generic error class, such as INVALID_SPARK_CONFIG for SparkConf.validateSettings.

Reply:
That's fine. Let me revert it out for now.